src/argilla_sdk/records/_resource.py

```diff
@@ -76,6 +76,12 @@ def __init__(
             _server_id: An id for the record. (Read-only and set by the server)
             _dataset: The dataset object to which the record belongs.
         """
+
+        if fields is None and metadata is None and vectors is None and responses is None and suggestions is None:
```

Comment:
```suggestion
    if all(x is None for x in [fields, metadata, vectors, responses, suggestions]):
        raise ValueError("At least one of fields, metadata, vectors, responses, or suggestions must be provided.")
```

Reply:
@burtenshaw did you miss this? or did not want to do this?

Reply:
@davidberenstein1957 Sorry not to mention. I'm not into the creation of a generator and data structure for a logical expression.